### DISCUSSION TITLE
Thermostat shows temperature in C. How to change to F?

### DISCUSSION THREAD

**Discussion by eigenphase** [Koenkk/zigbee2mqtt#14801] [Devices]
I got a Centralite (Pearl) Zigbee thermostat. I paired it with z2m but all numbers and settings are shown in C.

I have changed the setting on the device to F and I have confirmed it operates in F. Within z2m, in "Settings (specific)" I also selected "fahrenheit", multiple times, but no change.

Any advice appreciated.

**Comment by fitsou** [comment]
Hi there, unfortunately I cannot help you with your issue, but maybe you can help me with mine.
I'm trying to pair a Centraline Sensor but I'm unable to find the combination that will make it discoverable in MQTT...
I have manage to pair one after many tries but I didn't fully understand how (5 sec press of the Sensor button. 10 sec? continuing pressing?)

**Reply by eigenphase** [reply]
First, I have "fixed" the issue by setting imperial values in Home Assistant. z2m still shows Celsius but Home Assistant Fahrenheit.

Second, I can unfortunately not really help you. Pairing worked smoothly for me:

- First, it goes into pair mode automatically at the first start
- Second, I was very successful with unpairing and re-pairing by just following the instructions in the manual. This is a very weird menu to navigate but with manual it works out. If I recall correctly, un-pairing was 14-11 and re-pairing was 12-01.

You are asking about sensor, I have a thermostat. I assume pairing works differently here. For me, no need to push for a long time. Just entering the settings menu did do it.
